package-template/README.md

```diff
@@ -0,0 +1,27 @@
+This folder contains details about the template that should be used for each newly forked Pulsar Package repository.
+
+The files in this folder should be used as a loose to strict structure of new files to add to any given repo, with these files intending to accomplish the following:
+
+* Give proper attribution to the original maintainer of the forked package
+* Specify the level of support, and the expected method of interaction with the repository
+* The GitHub Action workflows to add to the forked repo to ensure a hands off approach is possible.
+
+With our goals stated, lets get to it, with the first template provided being below:
+
+---
+
+<!-- Pulsar Cooperative Package Repository Template, place at the top of the orignal README -->
```

Comment:
Just having a think about this. Do we want it added to the top of the Readme like this? This position of the readme should really be showing off the package itself. Not to discredit the original author but I think maybe this could be reduced to a hyperlink to an org document?

Reply:
Hmm. I honestly thought this was helpful within the readme itself to give it the best chance of being read, since in it, it gives credit to the author and sets expectations about the level of support.

If you feel a link does this just as well, I'm all for it, I just worry a link would be rather easy to miss completely by someone that doesn't understand how this works on this org

Reply:
@Daeraxa Now seeing your updated comment. I think you are right. It's too wordy. While I still feel like a link is easy to miss, that might be the best way to be unobtrusive about it

Reply:
What do we think about something like this?
![image](https://github.com/pulsar-cooperative/.github/assets/58074586/6c75249c-38d6-424c-b23e-4f30c4559831)
This is using the GitHub specific "Alerts" box which almost certainly won't work in the PPR or in Pulsar but I guess it could be done in HTML?

Reply:
You know, I think this is the right answer. And I'd be more than happy to add support for these alert boxes to the website, as it does use DFM anyway, might as well make sure it can support everything.

Great idea

Reply:
Well technically those aren't part of GFM (yeah I don't get it either) they are just extensions that GitHub uses for rendering Markdown. The issue is we would also need to be able to support that same style within Pulsar itself and not just in PPR.

Reply:
I soon realized this after the fact when I started looking up alerts (Since admittedly I wasn't familiar with them at all).

But yeah you are right, it's some extension on top of GFM, which makes sense, not really?

Anyways, I was thinking I could put together a `markdown-it` plugin to add support for these alerts, which means we simply use that plugin on PPR, and if memory serves `settings-view` is using `markdown-it` for displaying readmes? I know `settings-view` then goes on to use `marked` for the settings descriptions and such, but we should be good to go there

Reply:
I can totally live with that lol. Markdown *is* meant to be readable in plain text too so it isn't like it becomes invisible or incomprehensible by not being rendered, just untidy.

Reply:
Well I'm glad to see this may have given me my next Pulsar related project lol

Reply:
@Daeraxa